packages/flutter_tools/lib/src/resident_runner.dart

```diff
@@ -100,8 +100,11 @@ class FlutterDevice {
     if (flutterViews == null || flutterViews.isEmpty)
       return;
     for (FlutterView view in flutterViews) {
-      if (view != null && view.uiIsolate != null)
+      if (view != null && view.uiIsolate != null) {
+        // Manage waits specifically below.
+        // ignore: unawaited_futures
         view.uiIsolate.flutterExit();
+      }
     }
     await new Future<Null>.delayed(const Duration(milliseconds: 100));
```

Comment:
why 100ms? Why not 110ms? Seems like we'd be better off waiting all the futures here?